main.go

```diff
@@ -96,6 +96,10 @@ func Scrub(argument string) string {
 	u, err := url.ParseRequestURI(argument)
 	if err == nil && u.Host != "" && contains(allowedSchemes, u.Scheme) {
 		u.Scheme = "https"
+		// clear the user if there is no password, since it's common to use git@github.com
```

Comment:
Could you tweak this slightly to be more informative? For me, the "it's common" doesn't quite provide enough context... 

Maybe what you put in slack:
> So a ssh://git@github.com transforms to https://github.com/ because we happen to know MOST of the time the git is just a tell to the server you’re trying to do Git stuff and not actually trying to SSH into the server.